internal/cmd/flags.go

```diff
@@ -63,6 +64,10 @@ func (f *flags) bindCallTimeout(flagSet *pflag.FlagSet) {
 	flagSet.StringVar(&f.callTimeout, "call-timeout", "60s", "The maximum time to for all calls to be completed.")
 }
 
+func (f *flags) bindCertificateFile(flagSet *pflag.FlagSet) {
+	flagSet.StringVar(&f.certificateFile, "certificate", "", "SSL Certificate to make requests with.")
```

Comment:
Flag should either be named `--certificate-file` or variable should be named `certificate`.

Reply:
We could be even more specific and name this `ssl-certificate` & `sslCertificate`.

Reply:
I personally vote for `--ssl-certificate-path` to continue with this `ITEM-NOUN` naming scheme. @amckinney thoughts?

Reply:
either are fine by me, let's prefix with TLS instead of SSL, however